src/components/elementView/detailedView/assetView/assetView.scss

```diff
@@ -60,7 +62,7 @@ margin:auto;
 
   max-width:30pt;
   min-width:30pt;
-  border: 1px solid #BFCCD6 !important;
+  border: 1px solid #BFCCD6;
```

Comment:
is removing the `!important` tag to allow for highlighting of chassis diagrams?

Reply:
yes!